setup.py

```diff
@@ -74,9 +74,22 @@ def run(self):
         install.run(self)
 
 
+class LlvmliteClean(Command):
+    """Custom clean command to tidy up the project root."""
+    user_options = []
+    def initialize_options(self):
+        pass
+    def finalize_options(self):
+        pass
+    def run(self):
+        os.system('rm -vrf ./build ./dist ./ffi/*.so ./*.egg-info')
```

Comment:
This is really system-specific. If you want to trigger a rebuild on Unix, `touch ffi/*.cpp` should be enough. If we want a `setup.py clean` command, it should also work on Windows...
(also, distutils / setuptools has its own `clean` command, so we should probably derive from it)